examples/spring/pom.xml

```diff
@@ -91,6 +105,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
```

Comment:
there is no need for this plugin, lets move the IT classes under `test` directory

Reply:
Hi Dariusz,
I moved the IT classes under `test` directory.